backend/src/main/java/co/inventorsoft/scripty/validation/UpdatePasswordMatches.java

```diff
@@ -0,0 +1,27 @@
+package co.inventorsoft.scripty.validation;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Target({FIELD,TYPE, ANNOTATION_TYPE })
+@Retention(RUNTIME)
+@Constraint(validatedBy = UpdatePasswordMatchesValidator.class)
+@Documented
+public @interface UpdatePasswordMatches {
```

Comment:
I think @symynjuk has already created similar constraint. Please try to reuse it, or adjust it in a way it become reusable